recipes/types-aiobotocore-batch/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "types-aiobotocore-batch" %}
+{% set version = "2.3.4.post1" %}
+{% set lib_version = version.split(".post")[0] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ed2ee507f4638e0a375c2a4513fa34c5515cab13e0b979038addcff61beadf98
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - aiobotocore
+    - typing_extensions
+
+test:
+  imports:
+    - types_aiobotocore_batch
+  requires:
+    - pip
+    - boto3
```

Comment:
These packages are stubs-only, so `boto3` is generally not required. However, I will address it in the next version to make wrap these imports in `try-except` in case `boto3` is not available. Can we leave it as it is?